code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm

```diff
@@ -374,4 +374,36 @@
 	. = ..()
 	qdel(vat)
 
+/datum/micro_organism/cell_line/clown/fuck_up_growing(var/obj/machinery/plumbing/growing_vat/vat)
+	vat.visible_message("<span class='warning'>The biological sample in [vat] seems to have created something horrific!</span>")
+	QDEL_NULL(vat.biological_sample) //Kill off the sample, we're done
+
+	var/mob/selected_mob = pick(list(/mob/living/simple_animal/hostile/retaliate/clown/mutant/slow, /mob/living/simple_animal/hostile/retaliate/clown/mutant/blob/slow))
+
+	new selected_mob(get_turf(vat))
+
+/datum/micro_organism/cell_line/clown/bananaclown
+	desc = "Clown bits with banana chunks"
+
+	required_reagents = list(/datum/reagent/consumable/banana)
+	supplementary_reagents = list(/datum/reagent/consumable/banana_peel = 4)
```

Comment:
correct me if I'm wrong, but since none of these three have any inhibiting reagents, by the time that one of these clown cell lines finishes growing, all of the other clown cell lines in the vat will be at 30 growth or more, right? so there's always a risk of a vat ""accident""?

Reply:
fair point ill add some more supplements

Reply:
but honestly failing these cell lines sometimes by chance is fine as it still spawns mobs

Reply:
Yeah, I was thinking it was intentional for that reason, especially since bananaclowns (goes Z O O M and shits out bananas passively) and fleshclowns (has H A N D S) are so strong (longface sucks, though, even as a bruiser).